doc/policies/kernels.hpp

```diff
@@ -19,9 +19,8 @@ A kernel (or `Mercer kernel') \f$\mathcal{K}(\cdot, \cdot)\f$ takes two objects
 as input and returns some sort of similarity value.  The specific details and
 properties of kernels are outside the scope of this documentation; for a better
 introduction to kernels and kernel methods, there are numerous better resources
-available, including \ref
-http://www.eric-kim.net/eric-kim-net/posts/1/kernel_trick.html "Eric Kim's
-tutorial".
+available, including
+<a href="http://www.eric-kim.net/eric-kim-net/posts/1/kernel_trick.html>"Eric Kim's tutorial"</a>
```

Comment:
```suggestion
<a href="http://www.eric-kim.net/eric-kim-net/posts/1/kernel_trick.html>Eric Kim's tutorial</a>
```
I think, without using `\ref`, the quotes won't be needed. :+1: